data/_models/shared/Collections.csv

```diff
@@ -1,53 +1,51 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
-Collections,,,,,,,,,,,dcat:Resource,"Generic listing of all resources. Should not be used directly, instead use specific types such as Databanks and Studies","DataCatalogueFlat,EMA",
+Collections,,,,,,,,,,,dcat:DatasetSeries,"Generic listing of all resources. Should not be used directly, instead use specific types such as Databanks and Studies","DataCatalogueFlat,EMA",
 Collections,,overview,heading,,,,,,,,,General information,DataCatalogueFlat,
-Collections,,id,,1,true,,,,,,dcterms:identifier,Internal identifier,"DataCatalogueFlat,EMA",
-Collections,,pid,,2,,,,,,,dcterms:identifier,Persistent identifier,"DataCatalogueFlat,EMA",
-Collections,,acronym,,,,,,,,,dcterms:identifier,Acronym if applicable,"DataCatalogueFlat,EMA",
-Collections,,name,text,3,true,,,,,,"dcterms:title,foaf:name",Name used in European projects,"DataCatalogueFlat,EMA",
-Collections,,collection type,ontology,,,CatalogueOntologies,Collection types FLAT,,,,,,DataCatalogueFlat,
+Collections,,id,,1,true,,,,,,dct:identifier,Internal identifier,"DataCatalogueFlat,EMA",
```

Comment:
this is an internal id. I think we should just use the pid. pid is filled by UMCG research data catalogue, so we can use it.

Reply:
Can you elaborate on this one? `pid` is defined in the next column

Reply:
For 'id' we now use the cohort acronym for instance, which is not a persistent identifier. according to dcat-ap, dct:identifier should be a persistent identifier for the dataset.